avalon/tools/loader/widgets.py

```diff
@@ -309,6 +328,44 @@ def echo(self, message):
         print(message)
 
 
+class SubsetTreeView(QtWidgets.QTreeView):
+
+    def __init__(self, parent=None):
+        super(SubsetTreeView, self).__init__(parent=parent)
+        loading_gif = os.path.dirname(style.__file__) + "/svg/spinner-200.svg"
+        spinner = QtSvg.QSvgRenderer(loading_gif)
```

Comment:
My experience with svg in Qt is that it is slow in comparison to png or gif. Recommendation is to implement rotating QLabel with png, on the other side I guess that one svg won't affect Qt much.

Reply:
Yep, there's only one, and it's animation playback will be stopped once the loading is completed. So yeah, I think we are safe :P